scripts/process_twitter_data.py

```
@@ -0,0 +1,260 @@
+        
+        # Wait for server to start
+        for i in range(30):  # Wait up to 30 seconds
+            if check_server_running():
+                print("✅ HTTP server started successfully")
+                return True
+            time.sleep(1)
+            print(f"⏳ Waiting for server to start... ({i+1}/30)")
+        
+        print("❌ Server failed to start within 30 seconds")
+        return False
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)